domain/src/DomainNegotiatorInterface.php

```diff
@@ -46,6 +46,8 @@ public function getHttpHost();
 
   /**
    * Gets the id of the active domain.
+   *
+   * @return int|null|string
```

Comment:
Yeah - I saw that. The reason I did that was because DomainNegotiator::getActiveId() returns domain->id() which when I follow the train back, seems to be Entity->id() which has that signature. In truth, I don't know enough yet about the workings of the module to try to change that behavior so I thought documenting the exisiting possible return values was a sound starting point. I think the expectation is that id() returns a positive int, so changing DomainNegotiator::getActiveId() to "(int) return $this->domain->id();" and setting return type to int might be more in line with people's expectations.